libcnb/src/tracing.rs

```diff
@@ -139,28 +138,21 @@ impl<W: Write + Send + Debug> FileExporter<W> {
 }
 
 impl<W: Write + Send + Debug> SpanExporter for FileExporter<W> {
-    fn export(
```

Comment:
I was curious about what changed upstream to need this change, and I found:
https://github.com/open-telemetry/opentelemetry-rust/commit/dc9a5c87d67453e59f274f80335b3c18e08f78de

Looking at that, it seems the implementation can be simplified further by switching to native `async` syntax, so I copied the migration patterns seen in the upstream PR and pushed a commit with those changes:
https://github.com/heroku/libcnb.rs/pull/955/commits/afd2fe17ebfac85c63402247c8316604dcb1c499